### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by Baggz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "npm": "3.3.x"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com:apiaryio/refract-query.git"
+  },
+  "author": "Apiary Inc. <support@apiary.io>",
+  "license": "MIT",
+  "devDependencies": {
+    "babel": "",
```
Comment (freaz) [inline package.json:23]: Since it is OS library you should have properly specified versions.

**Comment by freaz** [comment]
One think which I would like to have it before merging is version in peckage.json

Other is just nitpick: missing new lines at the end of files.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "mocha": ""
+  },
+  "babel": {
+    "presets": [
+      "es2015",
+      "stage-0"
+    ]
+  },
+  "dependencies": {
+    "lodash": "^3.10.1"
```
Comment (XVincentX) [inline package.json:44]: Please do not take this as a dependency, but require a lodash instance.See [here](https://github.com/apiaryio/lodash-api-description/blob/master/lib/index.js#L4)

Furthermore (I am sorry to reiterate on this again), I really do not understand why this should be a separate repo and not included with the other mixins.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "refract-query",
+  "version": "0.0.1",
+  "description": "",
+  "main": "./lib",
+  "scripts": {
+    "prepublish": "not-in-install && npm run build || :",
+    "lint": "eslint src test",
+    "build": "babel src/ --out-dir lib/",
+    "test": "npm run lint && mocha --compilers js:babel-core/register -R spec --recursive"
```
Comment (pksunkara) [inline package.json:10]: Can we use peasant here? /cc @danielgtaylor

Comment (danielgtaylor) [inline package.json:10]: Yes we can definitely use [Peasant](https://github.com/danielgtaylor/peasant) here :+1:

**Pull request merged**